common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2702,17 +2703,28 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(512)
 
-  val CLIENT_PUSH_MAX_REQS_IN_FLIGHT: ConfigEntry[Int] =
-    buildConf("celeborn.client.push.maxReqsInFlight")
+  val CLIENT_PUSH_MAX_REQS_IN_FLIGHT_TOTAL: ConfigEntry[Int] =
+    buildConf("celeborn.client.push.maxReqsInFlight.total")
       .withAlternative("celeborn.push.maxReqsInFlight")
       .categories("client")
       .version("0.3.0")
-      .doc("Amount of Netty in-flight requests per worker. The maximum memory is " +
-        "`celeborn.client.push.maxReqsInFlight` * `celeborn.push.buffer.max.size` * " +
-        "number of workers * compression ratio(1 in worst case), say we have 50 workers," +
-        "the maximum memory is: 64KiB * 16 * 50 = 50MiB")
+      .doc("Amount of total Netty in-flight requests. The maximum memory is " +
+        "`celeborn.client.push.maxReqsInFlight.total` * `celeborn.push.buffer.max.size` " +
+        "* compression ratio(1 in worst case): 64KiB * 256 = 32MiB")
       .intConf
-      .createWithDefault(16)
+      .createWithDefault(256)
+
+  val CLIENT_PUSH_MAX_REQS_IN_FLIGHT_PERWORKER: ConfigEntry[Int] =
+    buildConf("celeborn.client.push.maxReqsInFlight.perWorker")
+      .categories("client")
+      .version("0.3.0")
```

Comment:
are you going to vote -1 for 0.3.0 rc1?

Reply:
I'm not sure yet, because the worst case should be very rare.

Reply:
we should mark it as 0.3.0 without `alternative` or 0.3.1 with `alternative`

Reply:
It's newly introduced from 0.3.0 and no ```alternative```

Reply:
Resolve this conversation since we are going to fail 0.3.0-rc1